README.md

```diff
@@ -1 +1,271 @@
+```
+
+### Configuring WebPack
+
+1. Initialize package.json:
+
+```sh
+npm init y
+```
+
+2. Install webpack:
+
+```sh
+npm install webpack webpack-cli --save-dev
+```
+
+3. Install CSS style loader:
+
+```sh
+npm install --save-dev style-loader css-loader
+```
+
+### Setup
+
+- Clone this [repository](https://github.com/M-Anwar-Hussaini/JavaScriptCapstone) to your desired folder:
+
+- Example commands:
+
+```sh
+  cd [YOUR FOLDER]
+  git clone https://github.com/M-Anwar-Hussaini/JavaScriptCapstone.git
+```
+
+### Install
+
+- Install this project by cloning or downloading the master branch of this [repository](https://github.com/M-Anwar-Hussaini/JavaScriptCapstone) and run `index.html` file on the root of repository.
+
+### Usage
+
+- To run the project, execute the following command:
+
+```sh
+ cd [YOUR FOLDER]
+ git clone https://github.com/M-Anwar-Hussaini/JavaScriptCapstone.git
+```
+
+### Run tests
+
+1. WebHint ☑
+
+```
+npx hint .
+```
+
+2. Stylelint ☑
+
+```
+npx stylelint "**/*.{css,scss}"
+```
+
+3. ESLint ☑
+
+```
+npx eslint .
+```
+
+### Deployment
+
+**This project is deployed by the author, no permission for deployment by any other client.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Mohammad Anwar Hussaini**
+
+- 👤 GitHub: [@Anwar Hussaini](https://github.com/M-Anwar-Hussaini)
+- 👤 Twitter: [@MAnwarHussaini](https://twitter.com/MAnwarHussaini)
+- 👤 LinkedIn: [Mohammad Anwar Hussaini](https://www.linkedin.com/in/mohammad-anwar-hussaini-876638267/)
+
+👤 **Sergio Usma**
+
+- GitHub: [@sergio-usma](https://github.com/sergio-usma)
+- Twitter: [@VonUsma](https://twitter.com/vonusma)
+- LinkedIn: [Sergio Andres Usma](https://www.linkedin.com/in/sergiousma/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Release Version]**
+- [ ] **[Host to a server]**
+- [ ] **[Use bootstrap framework]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/M-Anwar-Hussaini/JavaScriptCapstone/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, kindly drop a start for the [repository](https://github.com/M-Anwar-Hussaini/JavaScriptCapstone);
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+**I would like to thank the following individuals and organizations for their contribution to this project.**
+
+- 🙏 [**Microvere**](https://www.microverse.org/?grsf=mohammad-a-nbtazu) for its arrangement and supports to all Micronauts and me.
```

Comment:
I think you could add here something like:

_I would like to express my heartfelt gratitude to Microverse for the invaluable learning experience they have provided. The supportive community, dedicated mentors, and remote collaboration opportunities have enhanced my technical skills and prepared me for real-world projects. I extend my appreciation to the mentors and staff members for their guidance and support. The friendships and knowledge sharing within the Microverse community have made this journey truly rewarding._